R/epiprocess-data.R

```diff
@@ -66,87 +74,117 @@
+#'   These signals are taken directly from the JHU CSSE \href{https://github.com/CSSEGISandData/COVID-19}{COVID-19 GitHub repository} without changes.
+#' * Furthermore, the data has been limited to a very small number of rows,
+#'   formatted into a tibble, and the signal names slightly altered.
 "covid_incidence_outliers"
 
-#' COVID Cumulative Cases Example
-#'
-#' Data set for 4 states containing COVID-19 Cumulative Cases as reported
-#' by JHU-CSSE and downloaded from the CMU Delphi COVIDcast Epidata API.
-#' This data set covers the period from March 2020 to January 2022, and is
-#' used in the epiprocess Vignette on ... .
-#'
+#' Subset of COVID-19 Cumulative Cases from 4 states
+#' @description
+#' Data set for 4 states containing COVID-19 Cumulative Cases as reported by
+#' JHU-CSSE and downloaded from the CMU Delphi COVIDcast Epidata API. This
+#' data set covers the period from March 2020 to January 2022, and is limited
+#' to California, Florida, New York, and Texas.
+#'
+#' It is used in the {epiprocess} "Getting Started" vignette.
+#'
+#' @format An object of class \code{tbl_df} (inherits from \code{tbl}, \code{data.frame}) with 2808 rows and 15 columns.
+#' @section Data dictionary:
+#' A tibble with 2,808 rows and 15 variables:
+#' \describe{
+#'   \item{geo_value}{the geographic value associated with each row of measurements.}
+#'   \item{signal}{name of metric, derived from upstream data.}
+#'   \item{source}{name of upstream data source.}
+#'   \item{geo_type}{spatial resolution of the signal.}
+#'   \item{time_type}{temporal resolution of the signal.}
+#'   \item{time_value}{the time value associated with each row of measurements.}
+#'   \item{direction}{trend classifier (+1 -> increasing, 0 -> steady or not determined, -1 -> decreasing).}
+#'   \item{issue}{time unit (e.g., date) when the signal data were published.}
+#'   \item{lag}{time delta (e.g. days) between when the underlying events happened and when the data were published.}
+#'   \item{missing_value}{an integer code that is zero when the value field is present and non-zero when the data is missing (see \href{https://cmu-delphi.github.io/delphi-epidata/api/missing_codes.html}{missing codes}).}
+#'   \item{missing_stderr}{an integer code that is zero when the stderr field is present and non-zero when the data is missing (see \href{https://cmu-delphi.github.io/delphi-epidata/api/missing_codes.html}{missing codes}).}
+#'   \item{missing_sample_size}{an integer code that is zero when the sample_size field is present and non-zero when the data is missing (see \href{https://cmu-delphi.github.io/delphi-epidata/api/missing_codes.html}{missing codes}).}
+#'   \item{value}{Cumulative number of confirmed COVID-19 cases, derived from the underlying data source.}
+#'   \item{stderr}{approximate standard error of the statistic with respect to its sampling distribution, null when not applicable.}
```

Comment:
```suggestion
#'   \item{stderr}{Approximate standard error of the statistic with respect to its sampling distribution, NA when not applicable.}
```
or something more tailored to this object (stderr is all NAs)